QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+QUEST_JOBSTEP_20150317_000779	$And as he rode the arrow towards the moon, Rumpelstiltskin also realized the fact that the game cannot end.{nl}As time passed, the arrow's power subsided and began falling towards the ground, however, Lydia Schaffen shot another arrow and caused the arrow that Rumpelstiltskin rode on to stop still, floating in the sky.{nl}
+QUEST_JOBSTEP_20150317_000780	$Rumpelstiltskin thought he'd ask Lydia Schaffen what should he do to get out of his situation, but before he could, Lydia Schaffen began to talk,{nl}'For the game to end, you need to ride the arrow until it arrive at its target. The game cannot end before then. After you have touched the moon, and if I come back, then I will definitely accept the compensation.'{nl}After she spoke, Lydia Schaffen began walking along the forest road. Rumpelstiltskin looks as Lydia Schaffen walks away, and was unable to do anything special to change the course of the game.
+QUEST_JOBSTEP_20150317_000781	$According to the anecdote, Rumpelstiltskin has not reached the moon yet, and is still flying there with the magic in his body. {nl}It could also be the case that he returned before his demonic life span ended.{nl}
+QUEST_JOBSTEP_20150317_000782	$Nevertheless, as we know, Lydia Schaffen has passed away long ago. So while the demon did not lose the game, he has lost his chance to win the game forever. {nl}Still, the one matter left that we, as the current generation, are curious about is what the agreed compensation between Lydia Schaffen and Rumpelstiltskin was. {nl}Of course, we could ask a particular person, err, there is a thing that may still exist. However, as we do not know whether he has returned or not, it will be hard to settle this lingering matter for quite some time.{nl}{nl}
+QUEST_JOBSTEP_20150317_000783	$Atman Extension
+QUEST_JOBSTEP_20150317_000784	$Atman Extension
+QUEST_JOBSTEP_20150317_000785	$Congratulations on receiving this book, which no criminal may see. There are many ways to use magic, but the magic used by Linkers are a particular case. Other mages exercise effect-based powers rather than magic. That is to say, to fight an enemy with a fireball, or with lightning, does not truly bind the enemy with magic. 
+QUEST_JOBSTEP_20150317_000786	$These attacks only use magic to make a fireball and lightning. Then, the mages controls the magical energy's direction and destination, releasing it after settling on both.
+QUEST_JOBSTEP_20150317_000787	$In other words, these mages are unable to handle enemies without Linkers; as they need to channel and regulate surrounding energy, these mages do not have the time to easily handle the targeting their magic towards an enemy. However, as you have chosen to walk the path of the Linker, the way of your magic will differ completely. The way of a Linker's magic is by directly linking your magic with your opponent's magic, which includes searching for your target within seconds.
```

Comment:
"easily handle the targeting their magic" -> "easily handle targeting their magic"